src/test/java/TaskTest.java

```diff
@@ -0,0 +1,24 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskTest {
+    @Test
+    public void dummyTest() {
+        assertEquals(2, 2);
+    }
+
+    @Test
+    public void createTaskTest() {
+        String desc = "name_test";
+        Task sample = new Task(desc);
+        assertEquals(sample.getFileString(), "T|false|" + desc);
```

Comment:
The method `assertEquals(expected, actual)` is not used properly. Your parameters entered should be flipped around in terms of the order. 
i.e. `assertEquals("T|false|" + desc, sample.getFileString())`
Same goes for the other usage of `assertEquals(expected, actual)` in your code.